src/js/pensions/config/form.js

```diff
@@ -133,17 +138,62 @@ const formConfig = {
       }
 
     },
+    workHistory: {
+      title: 'Work history',
+      pages: {
+        disabilityHistory: {
+          title: 'Disability history',
+          path: 'disability/history',
+          depends: (formData) =>
+            moment().startOf('day').subtract(65, 'years').isBefore(formData.veteranDateOfBirth),
+          uiSchema: {
+            disabilities: {
+              'ui:title': 'What Disabilities prevent you from working?',
+              'ui:order': ['name', 'disabilityStartDate'],
+              'ui:options': {
+                viewField: DisabilityField
+              },
+              items: {
+                name: {
+                  'ui:title': 'Disability'
+                },
+                disabilityStartDate: dateUI('Date disability began')
+              }
+            },
+            // TODO: update schema with this field if stakeholders approve
```

Comment:
It's in progress, should be resolved after our first stakeholder review (in the next couple weeks).